converter/shieldhit/parser.py

```diff
@@ -165,47 +119,91 @@ def save_configs(self, target_dir: str):
             beam_f.write(str(self.beam_config))
 
         with open(path.join(target_dir, 'mat.dat'), 'x') as mat_f:
-            mat_f.write(str(self.mat_config))
+            mat_f.write(self.geo_mat_config.get_mat_string())
```

Comment:
Use `get_configs_json` to get filenames and file contents.
Putting filenames in multiple places violates DRY principle